changelog/next/bug-fixes/3578.md

```diff
@@ -0,0 +1 @@
+Having duplicate field names in `zeek-tsv` data no longer causes a crash, but rather errors out gracefully.
```

Comment:
Nitpick: We try to stick to the 80 chars limit here. But otherwise, this is good to go. Feel free to enable auto merge once you are ready.